apps/judicial-system/web/src/utils/stepHelper.tsx

```diff
@@ -190,6 +190,43 @@ const getAcceptingAlternativeTravelBanConclusion = (wc: Case): JSX.Element => {
   )
 }
 
+export const constructProsecutorDemandsAsString = (
+  workingCase: Case,
+  skipOtherDemands?: boolean,
+): string => {
+  return workingCase.requestedCustodyEndDate
+    ? `Þess er krafist að ${workingCase.accusedName}, kt. ${formatNationalId(
+        workingCase.accusedNationalId,
+      )}, sæti${
+        workingCase.parentCase &&
+        workingCase.parentCase?.decision === CaseDecision.ACCEPTING
+          ? ' áframhaldandi'
+          : ''
+      } ${
+        workingCase.type === CaseType.CUSTODY ? 'gæsluvarðhaldi' : 'farbanni'
+      } með úrskurði ${workingCase.court?.replace(
+        'Héraðsdómur',
+        'Héraðsdóms',
+      )}, til ${formatDate(
+        workingCase.requestedCustodyEndDate,
+        'EEEE',
+      )?.replace('dagur', 'dagsins')} ${formatDate(
+        workingCase.requestedCustodyEndDate,
+        'PPP',
+      )}, kl. ${formatDate(workingCase.requestedCustodyEndDate, TIME_FORMAT)}${
+        workingCase.requestedCustodyRestrictions?.includes(
+          CaseCustodyRestrictions.ISOLATION,
+        )
+          ? ', og verði gert að sæta einangrun á meðan á varðhaldi stendur.'
+          : '.'
+      } ${
+        workingCase.otherDemands && !skipOtherDemands
+          ? capitalize(workingCase.otherDemands || '')
+          : ''
+      }`
+    : 'Saksóknari hefur ekki fyllt út dómkröfur.'
+}
+
```

Comment:
This logic also exists in formatters.ts in the backend prject. Consider moving to libs/judicial-system/formatters and reusse here.

Reply:
Got a buunch of errors when I tried doing this. Do you know what's up? Maybe it's best to keep this as is?

<img width="549" alt="Screen Shot 2021-02-08 at 15 34 52" src="https://user-images.githubusercontent.com/3789875/107241856-4a6d2a80-6a23-11eb-805e-0938cb31df95.png">